Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -155,6 +164,59 @@ namespace Hazel {
 		std::chrono::time_point<std::chrono::steady_clock> m_StartTimepoint;
 		bool m_Stopped;
 	};
+
+	namespace InstrumentorUtils {
+
+		template <size_t N>
+		struct ChangeResult
+		{
+			char Data[N];
+		};
+
+		template <size_t N>
+		constexpr auto RemoveCdecl(const char(&expr)[N])
+		{
+			ChangeResult<N> result;
+
+			size_t srcIndex = 0;
+			size_t dstIndex = 0;
+			while (srcIndex < N - 2)
+			{
+				if (expr[srcIndex] == '_' &&
+					expr[srcIndex + 1] == '_' &&
+					expr[srcIndex + 2] == 'c' &&
+					expr[srcIndex + 3] == 'd' &&
+					expr[srcIndex + 4] == 'e' &&
+					expr[srcIndex + 5] == 'c' &&
+					expr[srcIndex + 6] == 'l' &&
+					expr[srcIndex + 7] == ' ')
```

Comment:
Can we align the indentation here?

Reply:
The intentation is only messed up here in github. Inside the file it is aligned as it should be.

Reply:
> The intentation is only messed up here in github. Inside the file it is aligned as it should be.

Ah yes, I forgot that we're using actual tabs here